lib/tagging/linkables.rb

```diff
@@ -0,0 +1,43 @@
+# Linkables
```

Comment:
If Linkables is reusable can we move it to gds-api-adapters or somewhere shared?